### DISCUSSION TITLE
Pagination - SEO

### DISCUSSION THREAD

**Discussion by FLMMFL** [GetPublii/Publii#1532] [Help]
Apologies in advance, if this is a really dumb question. I am getting a lot of duplication page titles and descriptions from an SEO report point of view, and the errors point to things like

/page/25/

/page/26/

/page/27/

/tags/hello-world/page/2/

/tags/hello-world/page/3/

/tags/hello-world/page/4/

I noticed in the publii settings that you can "disable homepage pagination"  and  "disable tags pagination indexation" and "disable tags pagination"  

Am i right in thinking that if I enable them and therefore disable the pagination, will that remove the above issue?  however if the pagination disappears, the user experience will suffer, as they will not see the fact that there are more pages, and also the search engines will not crawl the content on the other pages? which again sounds negative. Is there a better way to handle this...

**Comment by dziudek** [comment]
@FLMMFL - there is a big difference between "Disable X pagination indexing" and "Disable X pagination" where X is homepage/tags/authors

If you enable "Disable X pagination indexing" you will just add noindex,nofollow to all /page/* links and pagination will be still available, but for Google and other search bots it will be an information to not index these pages - what should in few weeks solve your issue.

If you enable "Disable X pagination" then you will totally remove /page/* items from your website what can affect UX of your website

**Reply by FLMMFL** [reply]
thank you , appreciate your pointers

**Comment by FLMMFL** [comment]
thank you for your help, so it seems  "Disable X pagination indexing"  is the only way forward to keep all the pages indexed in google and visible to visitors on the site. However in this scenario, if it will not index the content on  /2   /3   /4  pages, then does that not effect the overall searchability of the posts that sit on those /2   /3   /4   pages in search engines?

**Reply by dziudek** [reply]
@FLMMFL - it is a question for people who work with SEO not for me :) Technically posts are indexed also via sitemap and usually are available for some time on the first page. Pagination only adds additional links to this content. Also internal linking between posts is very important in this case
